debian_db_packages.lst

```diff
@@ -0,0 +1 @@
+postgresql
```

Comment:
@brousch I meant that redis could probably belong to the current dependency file - it's not exactly a conventional DB, and isn't used to actually store data at the moment, but what do you think?